### ISSUE TITLE
Create unit tests for each bot and suggest method

### ISSUE THREAD

**Issue by drittich** [TricksterCards/TricksterBots#18]
I don't see any unit tests in the solution. Is there another solution containing tests, or is this something yet to be done?

**Comment by johnson520** [comment]
We have tests in our main Trickster Cards repo. This is a great task for us to do. I've changed the title a bit.

**Comment by antross** [comment]
Quick update: we've made some initial progress here.
1.  Set up a new test project for this repo in https://github.com/TricksterCards/TricksterBots/commit/aec8331e8cf8ab7a78608a976c096637999a44c2 (including a few basic Spades tests).
2. Ported our Euchre bidding tests over in https://github.com/TricksterCards/TricksterBots/pull/20.
3. Added CI to this repo in https://github.com/TricksterCards/TricksterBots/pull/21 so tests automatically run for new changes.

Leaving this open as we continue to port the rest of our existing bot tests.
